pyiron_base/archiving/import_archive.py

```diff
@@ -62,19 +59,15 @@ def import_jobs(
     if compressed:
         extract_archive(archive_directory)
     archive_name = getdir(path=archive_directory)
-    if directory_to_import_to[-1] != '/':
-        directory_to_import_to = os.path.basename(directory_to_import_to)
-    else:
-        directory_to_import_to = os.path.basename(directory_to_import_to[:-1])
     # destination folder
-    des = os.path.abspath(os.path.join(os.curdir, directory_to_import_to))
+    des = project_instance.path
     # source folder; archive folder
-    src = os.path.abspath(os.path.join(os.curdir, archive_directory))
+    src = os.path.abspath(os.path.join(os.path.dirname(archive_directory), os.path.basename(archive_directory)))
```

Comment:
Isn't a `join` of `dirname` and `basename` identical to the input?
```suggestion
    src = os.path.abspath(archive_directory)
```

Reply:
I corrected this!